themes/digital.gov/layouts/partials/core/featured-communities.html

```diff
@@ -51,9 +51,12 @@ <h1>{{- .Title | markdownify -}}</h1>
         "image" .Params.dg_logo
         "alt" (printf "%s%s" .Params.dg_shortname " logo")
         "layout" $layout
+        "kicker" ""
```

Comment:
### Suggestion

If we don't use the kicker in this partial call, we can remove this line.